docs/pages/usage.md

```diff
@@ -0,0 +1,124 @@
+## Estimators
+
+`sknnr` provides estimators that are fully compatible, drop-in replacements for `scikit-learn` estimators:
+
+- [RawKNNRegressor](api/estimators/raw.md)
+- [EuclideanKNNRegressor](api/estimators/euclidean.md)
+- [MahalanobisKNNRegressor](api/estimators/mahalanobis.md)
+- [GNNRegressor](api/estimators/gnn.md)
+- [MSNRegressor](api/estimators/msn.md)
+
+These estimators can be used [like any other `sklearn` regressor](https://scikit-learn.org/stable/getting_started.html#fitting-and-predicting-estimator-basics):
+
+```python
+from sknnr import EuclideanKNNRegressor
+from sknnr.datasets import load_swo_ecoplot
+from sklearn.model_selection import train_test_split
+
+X, y = load_swo_ecoplot(return_X_y=True)
+X_train, X_test, y_train, y_test = train_test_split(X, y)
+est = EuclideanKNNRegressor(n_neighbors=3).fit(X_train, y_train)
+
+print(est.score(X_test, y_test)) 
+# 0.11496218649569434
+```
+
+In addition to their core functionality of fitting, predicting, and scoring, `sknnr` estimators offer a number of other features, detailed below.
```

Comment:
Did I forget anything that's worth mentioning?

Reply:
I think this is a great list.  Nothing else that I can think of needs to be added.